.drone.yml

```diff
@@ -329,7 +316,7 @@ steps:
       - name: mysql
         path: /var/lib/mysql
     settings:
-      filename: cache-staging-misc-build-artifacts.tar.gz
+      filename: cache-staging-misc-build-artifacts.with-node_modules.tar.gz
```

Comment:
Once this PR is approved, I'll switch this back to `cache-staging-misc-build-artifacts.tar.gz` and regenerate that build artifact with `node_modules` included before merging. I just didn't want to inflate the cache currently being used by other Drone runs with a fairly large directory until we're ready to use it.